src/main/java/com/emc/mongoose/storage/driver/pravega/PravegaStorageDriver.java

```diff
@@ -524,7 +523,11 @@ void submitEventReadOperation(final O evtOp, final String nodeAddr) {
       val endpointUri = endpointCache.computeIfAbsent(nodeAddr, this::createEndpointUri);
       val streamName = extractStreamName(evtOp.dstPath());
       scopeStreamsCache.computeIfAbsent(scopeName, ScopeCreateFunction::createStreamCache);
-      val readerGroup = UUID.randomUUID().toString().replace("-", "");
+      val readerGroupPart = (evtOp instanceof DataOperation) ? "DataOp" : "PathOp";
```

Comment:
operation will be always an instance of `DataOperation` since last changes regarding the byte streams support